examples/oauth/01-invoices.py

```diff
@@ -0,0 +1,26 @@
+# coding=utf-8
+
+from __future__ import print_function
```

Comment:
This file has no print() statements. Same goes for all other example files.

Reply:
Done